refined two manila modules (#564)
openstack-k8s-operators/data-plane-adoption: docs_user/assemblies/assembly_adopting-the-shared-file-systems-service.adoc

= Adopting the {rhos_component_storage_file}

The {rhos_component_storage_file_first_ref} in {rhos_long} provides a self-service API to create and manage file shares. File shares (or "shares"), are built for concurrent read/write access by any number of clients. This makes the {rhos_component_storage_file} essential in cloud environments that require a ReadWriteMany persistent storage.

> You access file shares in {rhos_acro} directly over a network. You must ensure that the networking in the {OpenStackShort} {rhos_prev_ver} environment matches the network plans for your new cloud after adoption. This ensures that tenant workloads remain connected to storage during the adoption process, even if the control plane has a minor interruption. The {rhos_component_storage_file} control plane services are not in the data path. Shutting down the API, scheduler, and share manager services do not impact access to existing shared file systems.

Author: {rhos_long} = Red Hat OpenStack Services on OpenShift 
{rhos_acro} = RHOSO
{OpenStackShort} = RHOSP

Are the sentences containing these attributes correct? I want to be sure I'm referring to the correct environment.

= Adopting the {rhos_component_storage_file}

> The {rhos_component_storage_file_first_ref} in {rhos_long} provides a self-service API to create and manage file shares. File shares (or "shares"), are built for concurrent read/write access by any number of clients. This makes the {rhos_component_storage_file} essential in cloud environments that require a ReadWriteMany persistent storage.

Reviewer: ```suggestion
The {rhos_component_storage_file_first_ref} in {rhos_long} provides a self-service API to create and manage file shares. File shares (or "shares"), are built for concurrent read/write access from multiple clients. This makes the {rhos_component_storage_file} essential in cloud environments that require a ReadWriteMany persistent storage.
```
I think "any number of" is colloquial.

You access file shares in {rhos_acro} directly over a network. You must ensure that the networking in the {rhos_prev_long} ({OpenStackShort}) {rhos_prev_ver} environment matches the network plans for your new cloud after adoption. This ensures that tenant workloads remain connected to storage during the adoption process, even if the control plane has a minor interruption. The {rhos_component_storage_file} control plane services are not in the data path. Shutting down the API, scheduler, and share manager services do not impact access to existing shared file systems.

> Typically, storage and storage device management networks are separate. Shared File Systems services only need access to the storage device management network.

Reviewer: ```suggestion
Typically, storage and storage device management are separate networks. Shared File Systems services only need access to the storage device management network.
```
